schema/accesskey.js

```diff
@@ -23,44 +39,128 @@ function AccessKey() {
         name: 'accesskey',
         required: {
             accesskeyid: 1,
-            accesskeysecret: 1
+            accesskeysecret: 1,
+            status: 1,
+            created: 1,
+            updated: 1,
         },
         optional: {
-            created: 1
+            description: 1,
+            // STS temporary credential fields
```

Comment:
Would it make sense to add some (all?) of these to the `READONLY_ATTRS` array?

Reply:
This prevents `if (operation === 'del') {` from ever being called, so probably not. Resolving.